biz.aQute.bndlib/src/aQute/bnd/osgi/Analyzer.java

```diff
@@ -2117,7 +2126,7 @@ void augmentImports(Packages imports, Packages exports) throws Exception {
 
 				String result = importAttributes.get(Constants.VERSION_ATTRIBUTE);
 				if ((result == null || !Verifier.isVersionRange(result))
-					&& complainAboutMissingVersionRange(packageRef, ee)) {
+					&& complainAboutMissingVersionRange(packageRef, ee) && !isUnstable) {
```

Comment:
what about passing `isUnstable` to the `complainAboutMissingVersionRange` method? so this method is the one deciding wether to complain or not. Maybe that would be the better place, as it also considers JDK-packages - JDK packages do not have a version which is fine.

Reply:
I would like to expose a different explanation for unstable APIs being referenced and for non-versioned or unknown dependencies being referenced.